app/code/core/Mage/Usa/Model/Shipping/Carrier/Usps.php

```diff
@@ -526,7 +526,6 @@ public function getCode($type, $code = '')
                 '0_FCLE' => Mage::helper('usa')->__('First-Class Mail Large Envelope'),
                 '0_FCL'  => Mage::helper('usa')->__('First-Class Mail Letter'),
                 '0_FCSL' => Mage::helper('usa')->__('First-Class Mail Stamped Letter'),
-                '0_FCP'  => Mage::helper('usa')->__('First-Class Package Service - Retail'),
```

Comment:
mmmm magento/magento2#37745 says that first class is replaced by ground advantage too. i've no way of verifying, the only feedback that I have is from a a customer and a user who write here and they're positive.

Reply:
You are right, the grace period of products

* First-Class Package Service
• First-Class Package Return Service
• Parcel Select Ground
• Ground Returns

will expire on September 30, 2023.

Reply:
I think this should be merged in order to be released in a couple of months realistically. who uses USPS (like a new customer of mine) already had to apply this patch or the USPS module was broken (since the ground shipping is usually the cheapest)